packages/validtion-errors/src/ValidationErrors.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import type * as shared from '@contentful/field-editor-shared';
```

Comment:
Could you use named imports here? We never used such imports in this repo and it would be nice to be consistent.